app.py

```diff
@@ -104,23 +111,24 @@ def __init__(self):
         self.public_key_bytes = self.public_key.public_bytes(
             encoding=serialization.Encoding.Raw, format=serialization.PublicFormat.Raw
         )
+        self.model_list = self.model_config.find_one({"name": "model_list"})["data"]
```

Comment:
Should we need to add a background process to update these kind of mongodb config for every 10 minutes example?

Reply:
That's a good idea.